contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -264,6 +264,26 @@ END;
 $BODY$
 LANGUAGE plpgsql STABLE;
 
+CREATE OR REPLACE VIEW sys.spt_tablecollations_view AS
+    SELECT
+        o.object_id         AS object_id,
+        o.schema_id         AS schema_id,
+        c.column_id         AS colid,
+        CASE WHEN p.attoptions[1] LIKE 'bbf_original_name=%' THEN CAST(split_part(p.attoptions[1], '=', 2) AS sys.VARCHAR(128))
```

Comment:
Have we verifed that typmod is indeed 128 from attribute info?

Reply:
The typmod in pg_attribute is in fact -1, so I have removed the typmod in the latest commit.